src/libs/Navigation/AppNavigator/ModalStackNavigators.js

```diff
@@ -532,6 +532,13 @@ const SettingsModalStackNavigator = createModalStackNavigator([
         },
         name: 'Settings_Wallet',
     },
+    {
+        getComponent: () => {
+            const SettingsWalletPage = require('../../../pages/settings/Wallet/WalletPage/WalletPage').default;
+            return SettingsWalletPage;
+        },
+        name: 'Settings_New_Wallet',
```

Comment:
Same here, use the `SCREENS.SETTINGS.WALLET` for the screen above

Reply:
Done